packages/component/src/Localization/getLocaleString.js

```diff
@@ -0,0 +1,15 @@
+export default function getLocaleString(value, language) {
+  const date = new Date(value);
+  const options = {
+    day: '2-digit',
+    hour: '2-digit',
+    hour12: false,
+    minute: '2-digit',
+    month: 'long'
+  };
+
+  if (window.Intl) {
+    return !!date && new Intl.DateTimeFormat(language, options).format(date);
```

Comment:
Can you check if playground is working properly? It broke on my box.

On my side, I am seeing `language === ''` while running in playground. And `new Intl.DateTimeFormat('', ...)` will throw an exception, while `new Intl.DateTimeFormat(undefined, ...)` won't.

I believe `language` if not defined, should be fallback `navigator.language` and ultimately fallback to `en-US`. Feels weird to me we are getting empty string here.